crates/driver/src/infra/blockchain/mod.rs

```diff
@@ -171,6 +172,7 @@ impl Ethereum {
                     value: Some(tx.value.into()),
                     data: Some(tx.input.clone().into()),
                     access_list: Some(tx.access_list.clone().into()),
+                    gas_price: self.simulation_gas_price().await,
```

Comment:
Does this need to be adjusted for Enso and Tenderly simulations as well? Otherwise we might have unexpected regressions when switching simulators.

Reply:
`temper` doesn't allow you to set the gas price in your request. Also we haven't used the tenderly simulator in forever because it became too expensive which is why I didn't implement it there initially. But it turned out that adding it there was not too much work.
But overall I'm inclined to get rid of tenderly given that we don't really use it.

Reply:
Didn't we turn it on during the nethermind outage?